conans/client/generators/pkg_config.py

```diff
@@ -34,12 +34,12 @@ def compiler(self):
     @property
     def content(self):
         ret = {}
-        for _, cpp_info in self.deps_build_info.dependencies:
-            ret["%s.pc" % cpp_info.name] = self.single_pc_file_contents(cpp_info)
+        for depname, cpp_info in self.deps_build_info.dependencies:
+            name = cpp_info.name.lower() if cpp_info.name != depname else depname
```

Comment:
so, if `cpp_info.name == depname` it won't go lowercase?

Reply:
that's what the logic says 😄

Reply:
yeah, but is it expected? AFAIK pkg-config file should be always in lower-case, right?

Reply:
I have seen that it is a very extended practice, but AFAIK pkg-config does not force the name to be lower-case

Reply:
checked:
http://manpages.ubuntu.com/manpages/cosmic/man5/pc.5.html
https://dev.gentoo.org/~mgorny/pkg-config-spec.html
there is no hard requirement for lower-case name